translations/README.bg.md

```diff
@@ -0,0 +1,86 @@
+# GitHub Резюме Генератор
+
+![React GitHub Resume](./src/assets/readme/screenshot.png)
```

Comment:
Please, change this route to `../src/assets/readme/screenshot.png`

Reply:
Fixing. FIY the original English readme uses `./src/assets/readme/screenshot.png`

Reply:
Sorry, it was a mistake. But the right is how I reviewed